source/documentation/tools/create-a-derived-table/instructions.md

```diff
@@ -419,7 +419,15 @@ Configurations are prioritised in order of specificity, which is generally the i
 
 ### Sources
 
-Sources are descriptions of the databases and tables already in Analytical Platform. With those tables defined as sources in dbt, it is then possible to select from source tables in your models using the [`source()`](https://docs.getdbt.com/reference/dbt-jinja-functions/source) function which helps define the lineage of your data. To see which sources have been defined, look in the [`./mojap_derived_tables/models/sources/`](./mojap_derived_tables/models/sources/) directory. If a database or table hasn't been defined as a source, let the data modelling team know at [#ask-data-modelling](https://asdslack.slack.com/archives/C03J21VFHQ9) and we'll look into adding it if it's appropriate to do so. Below is an example of using the `source()` function to select from the `contact` table in the `delius_prod` database:
+Sources are descriptions of the databases and tables already in Analytical Platform. With those tables defined as sources in dbt, it is then possible to select from source tables in your models using the [`source()`](https://docs.getdbt.com/reference/dbt-jinja-functions/source) function which helps define the lineage of your data. To see which sources have been defined, look in the [`./mojap_derived_tables/models/sources/`](./mojap_derived_tables/models/sources/) directory. If a database or table hasn't been defined as a source, then please follow below mentioned steps to get new source added.
+
```

Comment:
```suggestion
#### Adding a new source

If a database hasn't been defined as a source it will need to be added. Please follow the instructions below:

```